enterprise/api/v1/serializers.py

```diff
@@ -159,11 +162,16 @@ def validate_username(self, value):
         try:
             user = User.objects.get(username=value)
         except User.DoesNotExist:
+            error_message = ('[Enterprise API] The username {} does not exist'
```

Comment:
these messages are more specific, but keep in mind that splunk queries will need to be able to parse a standard format, so it would be good to add the detailed information in comma separated pairs after the specific message. Please see https://github.com/edx/ecommerce/pull/2248 for an example of what we're looking for wrt log messages.

Reply:
Messages modified